app/models/intake.rb

```diff
@@ -453,6 +453,7 @@ def filing_years
   end
 
   def most_recent_filing_year
+    # TODO: should the default be changed to 2020?
```

Comment:
Question for reviewer

Reply:
yes! I think this should be updated to 2020!